HumanResourceManagementSystem/src/main/java/com/HRmanagementsystem/dao/AdminDaoImpl.java

```diff
@@ -164,6 +164,30 @@ else if (res > 0 && accountActivation == 1)
         return messageString;
     }
 
+    /**
+     * Checks if a given username exists in the 'admin' database table.
+     *
+     * @param username The username to check for existence.
+     * @return {@code true} if the username exists in the database, {@code false} otherwise.
+     * @throws RuntimeException (implicitly) if a database error occurs (logged to console).
+     */
+    public boolean isUserExist(String username) {
+        boolean result = false;
+        try (Connection conn = DB_Connect.getConnection()) {
+
+            PreparedStatement ps = conn.prepareStatement("SELECT * from admin where username=? ");
+
+            ps.setString(1, username);
+
+            ResultSet rs = ps.executeQuery();
+            if (rs.next()) result = true;
+
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
```

Comment:
System.out.println(e.getMessage()); is a bad practice. Since this code changes is going to be a sample code for customer, I would suggest:
1. Re-throw it or update it;
2. Add a one line comment to make sure they handle exception properly according to their internal coding guideline.